### DISCUSSION TITLE
Sync favorite items with native iOS/Android Photo Library

### DISCUSSION THREAD

**Discussion by imlibra** [ente-io/ente#977] [Enhancements]
Google Photos on iOS have this feature
https://support.google.com/photos/answer/9845656

**Comment by ua741** [comment]
Hey @imlibra,
Thank you for suggesting this feature. It definitely make sense to provide this functionality. We will prioritise it and let you know once it's released.

**Reply by Zk2u** [reply]
Any news on this?
